cli/src/session.rs

```diff
@@ -97,22 +99,38 @@ impl Session {
             }
             let version = conn.version().await.unwrap_or_default();
             println!("Connected to {}", version);
-            println!();
 
+            let config = sled::Config::new().temporary(true);
+            let db = config.open()?;
+            // ast keywords
+            {
+                let keywords = all_reserved_keywords();
+                let mut batch = sled::Batch::default();
+                for word in keywords {
+                    batch.insert(word.to_ascii_lowercase().as_str(), "k")
+                }
+                db.apply_batch(batch)?;
```

Comment:
better to limit the size or here may stuck for a while to save keywords into disk.